mergify_engine/tests/functional/base.py

```diff
@@ -387,6 +389,15 @@ def setUp(self):
             self.r_o_integration.name,
         )
 
+        try:
+            self.cli_integration = github.get_client(
+                config.TESTING_ORGANIZATION, self.name, config.INSTALLATION_ID
+            )
+        except httpx.HTTPNotFound:
+            self.cli_integration = github.get_client(
+                config.TESTING_ORGANIZATION, self.name, config.INSTALLATION_ID
+            )
+
```

Comment:
why?

Reply:
The repo have just been created sometimes installation is not yet present.

Reply:
if that's good enough let's do this